src/ResourceManager/AzureBatch/Commands.Batch/Files/GetBatchNodeFileContentCommand.cs

```diff
@@ -83,10 +83,39 @@ public class GetBatchNodeFileContentCommand : BatchObjectModelCmdletBase
         [ValidateNotNullOrEmpty]
         public Stream DestinationStream { get; set; }
 
+        [Parameter(HelpMessage = "The start of the byte range to be downloaded. If this is not specified and ByteRangeEnd is, this defaults to 0.")]
+        [ValidateNotNullOrEmpty]
+        public long? ByteRangeStart { get; set; }
+
+        [Parameter(HelpMessage = "The end of the byte range to be downloaded. If this is not specified and ByteRangeStart is, this defaults to the end of the file.")]
+        [ValidateNotNullOrEmpty]
+        public long? ByteRangeEnd { get; set; }
```

Comment:
@matthchr does it make sense to use the `ValidateRange` attribute for these parameters so you can restrict the values passed through to these parameters? (_i.e.,_ negative values cannot be used)